packages/valory/connections/ledger/ledger_dispatcher.py

```diff
@@ -285,8 +291,12 @@ def get_transaction_receipt(
             and self.connection_state.get() == ConnectionStates.connected
         ):
             try:
-                transaction = api.get_transaction(
-                    message.transaction_digest.body, raise_on_try=True
+                transaction = await self.wait_for(
```

Comment:
why the change, the `get_transaction` remained unchanged.

Reply:
The method is async, get_transaction is sync. It can block the execution.